lib/common_test/test/ct_surefire_SUITE_data/skip_all_surefire_SUITE.erl

```diff
@@ -0,0 +1,57 @@
+%%--------------------------------------------------------------------
+%% %CopyrightBegin%
```

Comment:
remove that test suite. rename existing suite for skipping (generalize it).
adjust test specs accordingly

Reply:
this turned out to be more tricky. test suite names are used for defining expected events during verification.
when reusing existing test suite I got a conflict and test failures ...

Reply:
this is now fixed.